### PR TITLE
fix some comments and test cases

### PR DESCRIPTION
**Pull request by Kevin-GuanJian**
fix some comments and test cases

### CODE REVIEW

File: storage/prefix.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ import (
 )
 
 // PrefixConfig relate a string prefix to a config object. Config
-// objects include accounting, permissions, and zones. PrefixConfig
+// objects include accounting, permissions, users and zones. PrefixConfig
```
Comment (tamird) [inline storage/prefix.go]: s/users /users, /

File: storage/range.go [hunk 2]
```diff
@@ -1016,7 +1016,7 @@ func (r *Range) maybeGossipFirstRange() error {
 
 // maybeGossipConfigs gossips those configuration maps for which the supplied
 // function returns true and whose contents are marked dirty. Configuration
-// maps include accounting, permissions, and zones. The store is in charge of
+// maps include accounting, permissions, users and zones. The store is in charge of
```
Comment (tamird) [inline storage/range.go]: ditto

File: storage/range.go [hunk 3]
```diff
@@ -1157,7 +1157,7 @@ func (r *Range) maybeSetCorrupt(err error) error {
 
 // loadConfigMap scans the config entries under keyPrefix and
 // instantiates/returns a config map and its sha256 hash. Prefix
-// configuration maps include accounting, permissions, and zones.
+// configuration maps include accounting, permissions, users and zones.
```
Comment (tamird) [inline storage/range.go]: ditto

File: storage/store.go [hunk 4]
```diff
@@ -837,8 +837,8 @@ func (s *Store) RaftStatus(raftID proto.RaftID) *raft.Status {
 // BootstrapRange creates the first range in the cluster and manually
 // writes it to the store. Default range addressing records are
 // created for meta1 and meta2. Default configurations for accounting,
-// permissions, and zones are created. All configs are specified for
-// the empty key prefix, meaning they apply to the entire
+// permissions, users and zones are created. All configs are specified
```
Comment (tamird) [inline storage/store.go]: ditto

**Comment by Kevin-GuanJian** [comment]
updated as @tamird ‘s comments.

**Comment by MBerhault** [comment]
LGTM

**Comment by MBerhault** [comment]
Thanks for this. CLAhub is currently disabled, so we'll need to find a way for you to sign the CLA. I'll let @bdarnell chime in when he comes online.

**Comment by Kevin-GuanJian** [comment]
Thanks!

**Comment by bdarnell** [comment]
Since this is such a small change I'm going to merge it without the CLA for now; we'll ask you to officially sign once either CLAHub is fixed or we've moved to something else.

**Pull request merged**